client/my-sites/site-settings/podcasting-details/index.jsx

```diff
@@ -260,6 +266,30 @@ class PodcastingDetails extends Component {
 						onAddTermSuccess={ this.onCategorySelected }
 						height={ 200 }
 					/>
+					{ isDefaultCategorySelected && (
+						<Notice isCompact status="is-info">
+							<p>
+								{ translate(
+									'Using this category (the {{strong}}default category{{/strong}} for the site) is not recommended.',
+									{
+										components: {
+											strong: <strong />,
+										},
+									}
+								) }
+							</p>
+							<p>
+								{ translate(
+									'Try creating a category named {{strong}}Podcast{{/strong}} instead.',
```

Comment:
Is it an i18n standard to break strings into paragraphs? I feel like (especially in the case of a notice), this adds a good bit of whitespace, and doesn't make sense as separate paragraphs. If we're trying to have separate, re-usable translated strings, can they stay in a single paragraph?

Reply:
It also looked weird and not very readable to me to have this on one line, and the line break was at a weird spot.  But, as above, let's wait until we decide what to do about this issue, if anything.